pkg/apb/types.go

```diff
@@ -103,6 +104,14 @@ type ServiceInstance struct {
 	Parameters *Parameters `json:"parameters"`
 }
 
+//IsEqual - Compares the Id and Parameters of the instance to determine if SericeInstance is equal to the instance passed in.
+func (s ServiceInstance) IsEqual(si *ServiceInstance) bool {
+	if s.Id.String() != si.Id.String() {
+		return false
+	}
+	return reflect.DeepEqual(s.Parameters, si.Parameters)
+}
+
```

Comment:
While this is technically correct, it starts to set a precedence for having IsEqual methods on our structs. I'd prefer not to add it to the ServiceInstance.